src/plone/restapi/tests/test_documentation.py

```diff
@@ -23,8 +23,10 @@
 from plone.namedfile.file import NamedBlobImage
```

Comment:
Hmm - the `skipTest` for Plone >= 5.1 still doesn't get removed in this PR though, no? Is this intentional?

Reply:
no - I'll look at that now

Reply:
@lukasgraf what skipTest fo you mean? There is no such thing. 
There is 

```
    @unittest.skipIf(not PLONE5, 'Just Plone 5 currently.')
    def test_controlpanels_get_item(self):
```
but this is not related to 5.1 or not.

Reply:
[This one](https://github.com/plone/plone.restapi/blob/fb94ad786dfea7c3e9d949f5a2e9ebc0e7ea8dd5/src/plone/restapi/tests/test_documentation.py#L148-L149), and two more further down - those are the ones that skip the tests that previously failed with the `ReadConflict`, but only for Plone >= 5.1